explainer.md

```diff
@@ -557,10 +606,12 @@ interface VRDevice : EventTarget {
 
 dictionary VRSessionCreateParametersInit {
   boolean exclusive = true;
+  VRPresentationContext outputContext = null;
```

Comment:
That's my feeling too. It's easier to add the ability to reset it in the future than it is to revoke it if it proves problematic.